[0m20:15:06.577548 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071AD2AA10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071A82B190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071AD2AF10>]}


============================== 20:15:06.577548 | dff2d6a2-afde-4ea1-a107-828f40fef997 ==============================
[0m20:15:06.577548 [info ] [MainThread]: Running with dbt=1.8.2
[0m20:15:06.593174 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\huawei\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt init', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m20:15:13.833495 [debug] [MainThread]: Starter project path: C:\Users\huawei\anaconda3\Lib\site-packages\dbt\include\starter_project
[0m20:15:13.873114 [info ] [MainThread]: 
Your new dbt project "medical12" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m20:15:13.873114 [info ] [MainThread]: Setting up your profile.
[0m20:16:02.156865 [info ] [MainThread]: Profile medical12 written to C:\Users\huawei\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m20:16:02.172506 [debug] [MainThread]: Command `dbt init` succeeded at 20:16:02.156865 after 55.71 seconds
[0m20:16:02.172506 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071A743790>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071AB0F010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002071AD5F890>]}
[0m20:16:02.172506 [debug] [MainThread]: Flushing usage events
[0m20:16:17.894815 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739604D6D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739608D2D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739604FDD0>]}


============================== 20:16:17.894815 | b743666e-5436-4d05-b3e3-5c903333546d ==============================
[0m20:16:17.894815 [info ] [MainThread]: Running with dbt=1.8.2
[0m20:16:17.894815 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\huawei\\.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt run', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m20:16:17.894815 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\huawei\Desktop\week7\Kara-Solutions\Task2\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m20:16:17.910441 [debug] [MainThread]: Command `dbt run` failed at 20:16:17.910441 after 0.13 seconds
[0m20:16:17.910441 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739603A4D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739608E550>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001739608DD10>]}
[0m20:16:17.910441 [debug] [MainThread]: Flushing usage events
